tests/test_ecr/test_ecr_policy_validation.py

```diff
@@ -70,8 +70,7 @@ def test_validate_error_parse(policy):
 
     # then
     ex = e.value
-    assert ex.code == 400
-    assert ex.error_type == "InvalidParameterException"
+    assert ex.code == "InvalidParameterException"
```

Comment:
HTTP status codes are a serialization concern (and are contained in the AWS service models).  Moto service exceptions should no longer need to define them directly.